views/install.sh.erb

```diff
@@ -169,6 +169,14 @@ elif test "x$os" = "xAIX"; then
   platform="aix"
   platform_version="`uname -v`.`uname -r`"
   machine="powerpc"
+elif test -f "/etc/os-release"; then
+  . /etc/os-release
```

Comment:
yeah, i think that's probably the right thing to do because thats the natural, trivial and completely un-error prone way to parse the file. it just does have that one side effect that something in that file could obliterate internal state of the script...  as long as we've thought about that...